lib/core/scheduler.rb

```diff
@@ -189,5 +146,67 @@ def connector_registered?(service_type)
         false
       end
     end
+
+    def schedule_triggered?(scheduling_settings, identifier, last_synced)
```

Comment:
It looks like `last_synced` is not needed here because it's already presented in `scheduling_settings`

Reply:
The `last_synced` value is placed a little differently between `scheduling` and `custom_scheduling`. The `scheduling_settings` object here will be a little different depending on where it originated from.

```
# scheduling
{
  enabled: <boolean>,
  interval: <string>
}

# custom_scheduling
{
  enabled: <boolean>,
  interval: <string>,
  name: <string>,
  last_synced: <string>,
  configuration_overrides: <object>
}
```

This comment made me realise I left `last_synced` out of the `custom_scheduling` mappings update so I can see where the confusion would come from. I'll fix that in a separate PR.

Reply:
I've updated the connector protocol in this PR to include `custom_scheduling.*.last_synced`